### DISCUSSION TITLE
How do I combine a Quarto Website with Quarto Dashboard?

### DISCUSSION THREAD

**Discussion by franzeder** [quarto-dev/quarto-web#1034] [Q&A]
[Quarto](https://quarto.org/) is a very impressive and handy scientific publishing system. You can create really nice websites with it. Now, you can also create [Dashboards](https://quarto.org/docs/dashboards/) rather easily with Quarto.

But is there a way to combine both? So create a website and integrate a dashboard into this website as a subside (but with the same design)?

If so, how do you do it and are there any good practice examples?

**Comment by emilyriederer** [comment]
Nothing to add here, but I was looking into same and Googling this issue

Just dropping link to same question on [StackOverflow](https://stackoverflow.com/questions/78053114/how-do-i-combine-a-quarto-website-with-quarto-dashboard) where there is at least one solution for those interested
